migrate/troubleshooting.md

```diff
@@ -3,7 +3,7 @@ title: Troubleshooting known issues
 excerpt: Troubleshooting known issues in database migrations
```

Comment:
This page is mostly about migrations ([Migrate your data to Timescale / Troubleshooting](https://docs.timescale.com/migrate/latest/troubleshooting/)). Maybe the notes about the jobs would be better placed in 
https://docs.timescale.com/use-timescale/latest/user-defined-actions/troubleshooting/.

Reply:
:man_facepalming: 

Moved. Kept the fixes to backquoting `pg_dump` and similar words.